.gitignore

```diff
@@ -129,6 +129,9 @@ locale/*/*.flag
 3rdParty/linux/
 3rdParty/mac/
 3rdParty/buildCache/
+3rdParty/android*
```

Comment:
build dependencies for android should be inside `3rdParty/buildCache` directory

Reply:
But it not because the android scripts.
It create two folders that start with android:
3rdParty\android\.wget-hsts
3rdParty\android-ndk-r18b

Reply:
Android scripts by default use `prefix` that is targeted to buildcache folder. If it's not - then scripts should be fixed

Reply:
I think it not. That why it also install the other folders openssl-* and curl-*
And I think It good idea that the prefix should be 3rdParty/buildCache/